.github/workflows/nodejs.yml

```diff
@@ -28,6 +28,9 @@ jobs:
         with:
           useRollingCache: true
           install-command: npm ci --foreground-scripts
+      - name: Upgrade corepack
+        if: matrix.node-version != '14.x'
+        run: npm i -g corepack@latest
```

Comment:
1. `corepack` is [part of Node.js](https://github.com/nodejs/corepack), so we can trust it.
2. I think I can remove this from the workflow anyway, and it's leftover from something else.

Reply:
Oh, I remember now.  The problem is this: we need corepack v0.19.0, but that version doesn't work on Node.js v14.  Maybe it can just be an optional dep in the root workspace.  At the very least, it should be removed from `node/package.json`'s dev deps, because that version is broken.

`corepack` ships with Node.js, of course, but its version is tied to the version of Node.js (much like `npm`).  That means we'll either need to add it as a dependency somehow (preferred) _or_ do a global upgrade.  We can't just use the version that ships with whatever version of Node.js we're testing on because older versions are busted.

Reply:
I've removed this bit from the job and moved `corepack: 0.19.0` to `optionalDependencies` in the root monorepo.  It needs to stay there until we drop Node.js v14.x support.